Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -484,7 +488,7 @@ theorem _root_.Filter.Eventually.trans_isBigO {f : α → E} {g : α → F'} {k
 
 theorem _root_.Filter.Eventually.isBigO {f : α → E} {g : α → ℝ} {l : Filter α}
```

Comment:
I'm confused: isn't this theorem exactly the same as `IsBigO.of_norm_eventuallyLE`? If the reason to keep both is that they apply through dot notation in slightly different situations, can you add a docstring to each of them referring to the other one and explaining the difference?

Reply:
Yes, I failed to find this one, so added the new one. IMHO, the new name is more natural. I'll add docstrings.

Reply:
I'm fine with keeping both.